netlify.toml

```diff
@@ -0,0 +1,7 @@
+[build]
+publish = "/"
+command = "npm run build"
+
+[build.environment]
+  PYTHON_VERSION = "3.7"
```

Comment:
@k-nut Thank you for the explanation in the commit message.
@mojoaxel Can we please add a comment here to document why Python is added here?

Proposed comment text:

> Only needed for Netlify to successfully build because it's automatically picking up the requirements.txt file.